### PR TITLE
[tests-only][full-ci]Reuse the core code

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR makes use of the code already present in OC10 inorder to limit duplication

## Related Issue

- part of: 
- https://github.com/owncloud/ocis/issues/4509
- https://github.com/owncloud/QA/issues/766

## How Has This Been Tested?

- CI
- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/SpacesContext.php [hunk 1]
```diff
@@ -456,6 +433,8 @@ public function setUpScenario(BeforeScenarioScope $scope): void {
 		$this->webDavPropertiesContext = $environment->getContext('WebDavPropertiesContext');
 		$this->favoritesContext = $environment->getContext('FavoritesContext');
 		$this->checksumContext = $environment->getContext('ChecksumContext');
+		$this->filesVersionsContext = $environment->getContext('FilesVersionsContext');
+		// $this->webDav = $environment->getContext('WebDav');
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/SpacesContext.php:437]: delete?

**Review by phil-davis — APPROVED** [review]
Great to see so much code can be deleted.

**Review by ScharfViktor — APPROVED** [review]
nice) Thank you for cleaning up.

**Comment by ScharfViktor** [comment]
While writing new tests, I came across the following:

for example here https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/search.feature#L13 we set using  `spaces DAV path` after this line we create project space space and space content. 
before searching we set using `new DAV path`  https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/search.feature#L18

in any test, try adding the creation of a folder in an existing project space 
Result: folder created in `personal` space 

in test we should switch to `spaces DAV path` after that back to `new DAV path`  - It only became obvious to me after debugging

**Comment by SwikritiT** [comment]
Yes that is true we have to switch the `Dav` path depending on which endpoint we want to do certain operations in

**Pull request merged**